packages/back-end/src/api/openapi/payload-schemas/PostExperimentPayload.yaml

```diff
@@ -201,6 +210,12 @@ properties:
   regressionAdjustmentEnabled:
     type: boolean
     description: Controls whether regression adjustment (CUPED) is enabled for experiment analyses
+  sequentialTestingEnabled:
+    type: boolean
+    description: Only applicable to frequentist analyses
+  sequentialTestingTuningParameter:
+    type: number
+    description: 5000 is default
```

Comment:
Technically the org default is the default? IDK if that's a distinction worth making on this niche setting.